vegafusion-runtime/src/data/tasks.rs

```diff
@@ -245,14 +251,14 @@ lazy_static! {
     .collect();
 }
 
-const DATASET_CDN_BASE: &str = "https://cdn.jsdelivr.net/npm/vega-datasets";
-const DATASET_TAG: &str = "v2.2.0";
+const DATASET_BASE: &str = "https://raw.githubusercontent.com/vega/vega-datasets";
```

Comment:
duckdb  has trouble loading files from jsdelivr, more reliable to pull from GitHub